.github/workflows/stack-integration_tests.yml

```diff
@@ -653,3 +653,19 @@ jobs:
           docker rm `docker ps -aq` --force || true
           tox -e stack.test.course
           docker rm `docker ps -aq` --force || true
+
+      - name: Job Report Status
+        if: always()
+        uses: ravsamhq/notify-slack-action@v1
+        with:
+          status: ${{ job.status }}
+          notify_when: 'failure'
+          notification_title: '{workflow} has {status_message}'
+          message_format: '{emoji} *{workflow}* {status_message} in <{repo_url}|{repo}>'
+          footer: 'Linked to Repo <{workflow_url}|{repo_url}>'
```

Comment:
Maybe we also attach the run_url to either message or footer

Reply:
This is done..